example/example-model/src/main/java/uk/gov/gchq/palisade/example/rule/ZipCodeMaskingRule.java

```diff
@@ -37,21 +40,61 @@ private Employee maskRecord(final Employee maskedRecord) {
         String zipCodeRedacted = zipCode.substring(0, zipCode.length() - 1) + "*";
         address.setStreetAddressNumber(null);
         address.setStreetName(null);
-        address.setCity(null);
         address.setZipCode(zipCodeRedacted);
         return maskedRecord;
     }
 
+    private Employee maskWholeAddress(final Employee maskedRecord) {
+        maskedRecord.setAddress(null);
+        return maskedRecord;
+    }
+
+    private Boolean isManager(final Manager[] managers, final UserId userId) {
```

Comment:
Can you refactor this isManager() method into an EmployeeUtils class rather than having it in two rules